index.html

```diff
@@ -1,116 +1,176 @@
-            <time datetime="2021-01-28">28 января</time>
-        </li>
-        <li class="news-item">
-            <p>В нашей команде пополнение Борис "ПуджХукер228" Стригунец, обладатель множества титулов и наград пополнил наши стройные ряды</p>
-            <time datetime="2021-01-28">28 января</time>
-        </li>
-    </ul>
-    <a class="button" href="news.html">Все новости о чоуках на чике</a>
-</section>
-<section class="gallery">
-    <h2>Фотогалерея</h2>
-    <figure class="gallery-section">
-        <a href="хз какая ссылка будет"><src="img/studio.jpg" width="286" height="164" alt="Интерьер"></src></a>
-    </figure>
-    <span class="button gallery-button gallery-button-back">Назад</span>
-    <span class="button gallery-button gallery-button-next">Рузке Вперде</span>
-</section>
-<section class="contacts">
-    <h2>Контактная информация</h2>
-    <p>Тут инфа<br>
-    ещё инфа<br>
-    инфа<br>
-    инфа<br>
-    инфа
+                <li class="feature-item">
+                    <h3>Дорого
+                    </h3>
+                    <p>Наши мастера - профессионалы своего дела и не могут стоить дешево.К тому же, разве цена не дает определенный статус?
+                    </p>
+                </li>
+                </li>
+            </ul>
+        </section>
+
+        <section class="news">
+                    <h2>Новости
+                    </h2>
+            <ul class="news-list">
+                <li class="news-item">
+                    <p>Нам наконец завезли Ягермейстер! Теперь Вы сможете пропустить стаканчик во время стрижки                        
+                    </p>
+                        <time datetime="2021-01-28">28 января                            
+                        </time>
+                </li>
+                
+                <li class="news-item">
+                    <p>В нашей команде пополнение Борис "ПуджХукер228" Стригунец, обладатель множества титулов и наград пополнил наши стройные ряды                        
+                    </p>
+                        <time datetime="2021-01-28">28 января
+                        </time>
+                </li>
+            </ul>
+                    <a class="button" href="news.html">Все новости о чоуках на чике
+                    </a>
+        </section>
+
+        <section class="gallery">
+                    <h2>Фотогалерея
+                    </h2>
+            <figure class="gallery-section">
+                    <a href="хз какая ссылка будет"><src="img/studio.jpg" width="286" height="164" alt="Интерьер">
+                                                    </src>
+                    </a>
+            </figure>
+                        <span class="button gallery-button gallery-button-back">Назад
+                        </span>
+                        <span class="button gallery-button gallery-button-next">Рузке Вперде
+                        </span>
```

Comment:
опять уровень вложенности смотри, они на одном уровне с фигурой
плюс это же кнопки, для кнопок есть тег
почему такие классы дал?
это не плохо, но для чего они были?
(тип учись обосновывать решения, почему именно это лучший выбор в данной ситуации, подчеркивая, в ДАННОЙ ситуации)

```suggestion
            </figure>
            <button class="button gallery-button gallery-button-back">
                Назад
            </button>
            <button class="button gallery-button gallery-button-next">
                Рузке Вперде
            </button>
```

Reply:
Cause of there is могут быть не только кнопки, которые будут для галлереи использоваться. Вдруг со временем нужны будут кнопки для других блоков

Reply:
все что нажимается, это либо кнопка, либо ссылка
на другую страницу оно не ведет, значит это кнопки